server/models/loaders/AtLoader.js

```diff
@@ -1,16 +1,10 @@
 const { getAts } = require('../services/AtService');
```

Comment:
Was there a reason for removing the singleton implementation on the `AtLoader` and `BrowserLoader`?

Reply:
Hi Erika! Good question, take a look at a "little essay" I wrote about it here: https://github.com/w3c/aria-at-app/pull/951#discussion_r1529061043